src/pages/profile/EditProfile.tsx

```diff
@@ -1,11 +1,104 @@
-import {ReactElement} from "react";
+import { ReactElement, useEffect } from "react";
+import { useMyDetailsQuery } from "@/entities/profile/api/profileApi.ts";
+import { ProfileLoader } from "@/widgets/Profile/ProfileInfo";
+import { EditProfileForm } from "@/features/editProfile/ui/EditProfileForm";
+import Svg from "@/shared/ui/Svg";
+import { FormProvider, useForm } from "react-hook-form";
+import {
+  EditProfileValidationSchema,
+  editProfileFormSchema,
+} from "@/features/editProfile/model/editProfileFormSchema";
+import { zodResolver } from "@hookform/resolvers/zod";
+import { EditProfileInfo } from "@/features/editProfile/ui/EditProfileInfo";
+import { defaultProfileFormValues } from "@/features/editProfile/model/constants";
+import { removeProfileExtraFields } from "@/features/editProfile/model/removeProfileExtraFields";
+import { useGetCategoriesQuery } from "@/features/searchFilter/api/categoriesApi";
+import { EditOptions } from "@/features/editProfile/ui/EditOptions";
+import { PageTitle } from "@/widgets/PageTitle";
 
 function EditProfile(): ReactElement {
-  return (
-    <section className="w-full max-w-[1215px] mx-auto pb-[98px] flex flex-row flex-wrap min-h-[1000px]">
-      <p>edit profile</p>
-    </section>
-  )
+  const {
+    data: profileData,
+    isLoading: isProfileDataLoading,
+    isError: isErrorProfileData,
+    error: errorProfileData,
+    isSuccess: isProfileDataSuccess,
+  } = useMyDetailsQuery();
+
+  const {
+    data: categories = { results: [] },
+    isSuccess: isCategoriesSuccess,
+    isError: isCategoriesError,
+    error: categoriesError,
+  } = useGetCategoriesQuery();
+
+  isErrorProfileData &&
+    console.log(
+      `Ошибка при получении данных пользователя - ${JSON.stringify(
+        errorProfileData
+      )}`
+    );
+
+  const isFormLoading = isProfileDataLoading || !isProfileDataSuccess;
+
+  const methods = useForm<EditProfileValidationSchema>({
+    resolver: zodResolver(editProfileFormSchema),
+    mode: "onBlur",
+    defaultValues: defaultProfileFormValues,
+  });
+
+  useEffect(() => {
+    if (isProfileDataSuccess) {
+      const editFormValues = removeProfileExtraFields(profileData);
+      methods.reset(editFormValues);
+    } else {
+      methods.reset(defaultProfileFormValues);
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [isProfileDataSuccess]);
+
+  if (isProfileDataLoading) {
+    return (
+      <div className="m-auto">
+        <ProfileLoader />
+      </div>
+    );
+  }
+
+  if (isProfileDataSuccess && isCategoriesSuccess) {
```

Comment:
error handling should be done in EditProfileForm so that the PageTitle is not hidden until the server responds